docs/sources/http_api/folder.md

```diff
@@ -19,6 +19,7 @@ The uid can have a maximum length of 40 characters.
 
 The General folder (id=0) is special and is not part of the Folder API which means
 that you cannot use this API for retrieving information about the General folder.
+It also means that folder name "General" and uid "general" are reserved, so you cannot use them when creating folders.
```

Comment:
Same comment here. Why do we need to include information about a General folder that the user cannot create?

Reply:
I don't think you need to include this statement because you've already covered it in the previous docs.